app/helpers/application_helper/button/collect_logs.rb

```diff
@@ -0,0 +1,8 @@
+class ApplicationHelper::Button::CollectLogs < ApplicationHelper::Button::Basic
+  def calculate_properties
+    super
+    if @record.try(:log_file_depot).try(:requires_support_case?)
+      self[:prompt] = true
```

Comment:
should this be `self["prompt"]` to match the other hash keys which all seem to be strings?